utils/mstfwmanager/patches/0002-configure-remove-check-for-std-regex-compatibility.patch

```diff
@@ -0,0 +1,51 @@
+From 05dc7c55e0653542f92ed4e791f5eecb0e83b99c Mon Sep 17 00:00:00 2001
+From: Til Kaiser <mail@tk154.de>
+Date: Wed, 3 Jan 2024 17:58:41 +0100
+Subject: [PATCH] configure: remove check for std::regex compatibility
+
+The configure script compiles and runs a simple program to check
+for std::regex compatibility. Remove the check since this
+isn't possible when cross-compiling and returns an error message.
```

Comment:
Report this to upstream or send this to upstream what they will say about this.

Reply:
Ok, I see that you were trying to solve this with upstream, which is nice and appreciated.

Anyway, could you please use the patch ,which is included in upstream?
https://github.com/Mellanox/mstflint/pull/871/files